app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +89,11 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [interpolated(params)['response'] || 'Event Created', code]
+      if interpolated['response_headers'].presence
```

Comment:
It's not strictly needed, but because the WebhookAgent spec has the expected response array hardcoded in many tests, putting this conditional here keeps all of them passing without changing a bunch of specs to include `, "text/plain", nil]`. Can see it both ways but this seemed slightly safer to me.